daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -117,6 +116,7 @@ import org.apache.daffodil.xml.QName
 import org.apache.daffodil.xml.RefQName
 import org.apache.daffodil.xml.XMLUtils
 import org.apache.daffodil.xml.DFDLCatalogResolver
+// import org.apache.daffodil.processors.HasSetDebugger
```

Comment:
I would go further and remove the source file defining HasSetDebugger:

daffodil-lib/src/main/scala/org/apache/daffodil/processors/Runtime.scala

Once nothing uses a piece of code anymore, it becomes dead code and should be removed.

Reply:
@tuxji I did not delete `HasSetDebugger` as it still is used by `daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala`. The `setDebugger` and `setDebugging` methods were set as deprecated here that is why they weren't delete but this can be done if it is something we want to do.

Reply:
I looked at TDMLRunner.scala and I don't think it has any good reason why DFDLTestSuite needs to extend HasSetDebugger.  If you remove the import, the extends HasSetDebugger, and the override keyword from the `setDebugger` and `setDebugging` methods and you still can build the code and run the tests without errors, we'll know for sure.

Reply:
Makes sense, doing this worked fine for the tests.